src/Mails_PT.php

```diff
@@ -177,4 +206,44 @@ private function addRecipientGroupFields( string $group, $cmb ) {
 		) );
 	}
 
+	//Create a custom HTML cmb Field for All variables
+	public function custom_variables_field($cmb, $variables)
+	{
+		$cmb->add_field(array(
+			'name' => 'Mail Variables',
+			'id'   => self::POST_TYPE_NAME . "mail_variables",
+			'type' => 'text',
+			'variables' => $variables,
+			'render_row_cb' => function ($field_args, $field) {
+				$id          = $field->args('id');
+				$label       = $field->args('name');
+				$variables = $field->args('variables');
```

Comment:
All of that is no longer needed when using a new cmb2 box. There should be no need for another field